### PR TITLE
Release polish

### PR DESCRIPTION
**Pull request by christianbaroni**
## What changed (plus any additional context for devs)
- 1559 improvements, UI cleanup
- improved gas input tap zones
- better errors/warnings
- fixed sheet resizing/animation issues
- removed gwei subtitle for custom in the context menu unless custom is selected

## PoW (screenshots / screen recordings)
Screen Shot 2021-12-13 at 8 04 57 AM

## Dev checklist for QA: what to test
- check nothing is broken on android

## Final checklist
[ ] Assigned individual reviewers?
[ ] Added labels?

### CODE REVIEW

**Review by estebanmino — APPROVED** [review]
code LGTM, also built it locally. I think e2e will fail, I can take a look to that

File: src/components/expanded-state/custom-gas/FeesPanel.js [hunk 1]
```diff
@@ -512,27 +525,25 @@ export default function FeesPanel({
...
-      // there's an e2e modifying this panel so I needed values that aren't dependant on the network conditions
-      const maxBaseFeeToValidate = IS_TESTING === 'true' ? 100 : currentBaseFee;
       if (
         greaterThan(
           multiply(MAX_BASE_FEE_RANGE[0], maxBaseFeeToValidate),
           maxBaseFee
         )
       ) {
         setMaxBaseFeeWarning(LOWER_THAN_SUGGESTED);
-      } else if (
```
Comment (estebanmino) [inline src/components/expanded-state/custom-gas/FeesPanel.js:529]: did you remove this warning on purpose?

**Review by jinchung — APPROVED** [review]
🌮

**Review by skylarbarrera — APPROVED** [review]
lgtm post CI, will address any android issues separately

**Comment by jinchung** [comment]
lgtm post CI also - will do a couple code cleanup things in FeesPanel separately

**Comment by christianbaroni** [comment]
@estebanmino merge commit broke a few things, taking a look

**Pull request merged**